pkg/cache/v3/delta.go

```diff
@@ -51,20 +51,21 @@ func createDeltaResponse(req *DeltaRequest, st *stream.StreamState, snapshot Sna
 	toRemove := make([]string, 0)
 
 	// If we are handling a wildcard request, we want to respond with all resources
-	if st.IsWildcard {
+	switch {
+	case state.IsWildcard():
```

Comment:
any reason to use a switch here? I would think a regular if/else would be more idiomatic